pxemgr/pxemanager.go

```diff
@@ -212,6 +214,9 @@ func (mgr *pxeManagerT) startIPXEserver() error {
 	mgr.router.Methods("PUT").PathPrefix("/admin/host/{serial}/set_etcd_cluster_token").HandlerFunc(withSerialParam(mgr.setEtcdClusterToken))
 	mgr.router.Methods("PUT").PathPrefix("/admin/host/{serial}/override").HandlerFunc(withSerialParam(mgr.override))
 
+	mgr.router.Methods("GET").PathPrefix("/admin/mayu_config").HandlerFunc(mgr.getMayuConfig)
+	mgr.router.Methods("PUT").PathPrefix("/admin/mayu_config").HandlerFunc(mgr.setMayuConfig)
```

Comment:
E.g. you had a quite strong opinion on having a slash at the end of each path, which I still not agree with, but that is another story. `mayu_config` in this case with underscore is also new to me. I did not see that before in our APIs. So it is a question to which extend we want to comply to our own rules. IMO we should apply all of your conventions to all of our projects. When we go with mayu in the long run, mayu should not be excluded.